pygeoapi/plugin.py

```diff
@@ -43,6 +43,7 @@
         'GeoPackage': 'pygeoapi.provider.geopackage.GeoPackageProvider',
         'OGR': 'pygeoapi.provider.ogr.OGRProvider',
         'PostgreSQL': 'pygeoapi.provider.postgresql.PostgreSQLProvider',
+        'Mongo': 'pygeoapi.provider.mongo.MongoProvider',
```

Comment:
For consistency, can / should this be called `'MongoDB': 'pygeoapi.provider.mongo.MongoProvider'`

Reply:
Changed.